api/skaffold.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: skaffold/v1beta13
 kind: Config
 build:
   artifacts:
-    - image: keptn/api
+    - image: keptn/api-service
```

Comment:
Same here, let's make sure we actually want that image name

Reply:
As discussed, let's stay with the image name keptn/api.